siemens/siemens_common.mk

```diff
@@ -201,13 +201,27 @@ batch: $(PARAMETER_DONE) $(PRESIM_GOAL) $(presim_hook)
 	     echo -e "$(GREEN)# Simulation successful $C"; \
 	 fi;
 
+## Run autocheck in console only
+.PHONY: autocheck_batch
+autocheck_batch: $(AC_DONE)
+	@-$(HDL_BUILD_PATH)/siemens/ac_pretty.sh '$(AC_REPORT)';
+.PHONY: ac_batch
+ac_batch: autocheck_batch
+
+## Run autocheck GUI
+.PHONY: autocheck
+autocheck: autocheck_batch
+	@echo -e "$O Starting autocheck GUI $C (see $(BLOG_DIR)/autocheck.log)"
```

Comment:
This command doesn't really write anything to `autocheck.log` does it? I think the previous "see log" prints should suffice.